static/app/views/performance/mobile/components/screensTable.tsx

```diff
@@ -63,7 +63,7 @@ export function ScreensTable({
     if (data.meta.fields[column.key] === 'percent_change') {
       return (
         <PercentChangeCell
-          deltaValue={parseFloat(row[column.key] as string)}
+          deltaValue={parseFloat((row[column.key] as string) ?? 0)}
```

Comment:
I think you might want `||` instead of `??` because `NaN ?? 0` still gives you `NaN`.